pages/common/josm.md

```diff
@@ -0,0 +1,32 @@
+# josm
+
+> Extensible OpenStreetMap editor for Java 8+.
+> More information: <https://josm.openstreetmap.de/>.
+
+- Launch JOSM:
+
+`josm`
+
+- Launch JOSM in maximized mode:
+
+`josm --maximize`
+
+- Launch JOSM and set a specific language:
+
+`josm --language {{de}}`
+
+- Launch JOSM and reset all preferences to default values:
```

Comment:
```suggestion
- Launch JOSM and reset all preferences to default:
```

I don't like that usage of values however we'll see what others think

Reply:
How about
```suggestion
- Launch JOSM and reset all preferences to their default values:
```